packages/ace-core-legacy/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ace-core",
+  "name": "ace-core-legacy",
```

Comment:
the `ace-core` name was intentional, as it's how the package was release [on npm](https://www.npmjs.com/package/ace-core) originally… I don't know if this matters a lot now that we move to the `@daisy` org for all our packages.

Reply:
Good point. Reverting this change. Thanks!

Reply:
Done: https://github.com/daisy/ace/pull/230/commits/a78db166ba5024cea745fbdce61068253de5df79